tests/translator/input/api_with_propagate_tags.yaml

```diff
@@ -0,0 +1,17 @@
+Parameters:
+  TagValueParam:
+    Type: String
+    Default: value
+
+Resources:
+  MyApiWithPropagateTags:
```

Comment:
I don't think this template results in all the generated resources from [here](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-specification-generated-resources-api.html) being created right? We should have a test that has all generated resources and make sure the tags propagate correctly.

Reply:
I have a complete set of tests in the upcoming PR which includes function/sfn. Will ignore this comment for now.